### PR TITLE
Added faq page

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

This PR adds new FAQ page for Eth Flow and also links for that page in header dropdown and FAQ sidebar

Снимак екрана из „2022-12-16 16-28-07“

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey Nenad! Great job. Some nitpicks: 
1. I think that colon is missing here:
colon
2. And the 2nd quotes is missing here:
2nd
3. I'd write this in the separate row and make 'Note' in bold
note from the new line
4. There is only 1 place in the section where we have periods in the end of listing:
image


Thanks!

File: src/cow-react/constants/routes.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export enum Routes {
   FAQ_TRADING = '/faq/trading',
   FAQ_AFFILIATE = '/faq/affiliate',
   FAQ_LIMIT_ORDERS = '/faq/limit-order',
+  FAQ_ETH_FLOW = '/faq/eth-flow',
```
Comment (alfetopito) [inline src/cow-react/constants/routes.ts:22]:
```suggestion
  FAQ_ETH_FLOW = '/faq/sell-native',
```

**Review by alfetopito — APPROVED** [review]
Only a suggestion to use a different URL path than the internal name

**Comment by nenadV91** [comment]
@elena-zh Should be fixed now

**Review by anxolin — APPROVED** [review]
Slight preference, but i feel the title in the menu is too complicated.

image

**Review by elena-zh — APPROVED** [review]
LGTM!

**Pull request merged**